src/Native/pkg/dir.props

```diff
@@ -22,4 +31,13 @@
     <Ubuntu1604NativePath Condition="'$(Ubuntu1604NativePath)' == ''">$(BuildNativePath)</Ubuntu1604NativePath>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
   </PropertyGroup>
+
+  <!-- Add path globs specific to native binaries to exclude unnecessary files in packages. -->
+  <ItemGroup>
+    <AdditionalLibPackageExcludes Include="%2A%2A\%2A.dbg" Condition="'$(OSGroupPath)'=='Linux'" />
```

Comment:
Can we just use $(SymbolFileExtension) here instead (and have the glob be the same?)